src/code/fault_ootn64.c

```diff
@@ -0,0 +1,835 @@
+#include "global.h"
+#include "terminal.h"
+#include "alloca.h"
+
+typedef struct FaultMgr {
+    OSThread thread;
+    char unk_1B0[0x400];
+    OSMesgQueue queue;
+    OSMesg msg;
+    s32 framebuffer;
+    u16 fbWidth;
+    u16 fbDepth;
+    FaultClient* clients;
+} FaultMgr; // size = 0x5D8
+
+struct FaultCursorCoords {
+    s32 x;
+    s32 y;
+}; // size = 0x8
+
+u32 sFaultDrawerFont[] = {
```

Comment:
I in fact did not check entirely and just copypasted from existing fault.c, checking just a few values

I don't know if / how much to share with fault.c (also applies to some other comments)

MM recently made this blob extracted instead of committed I think

Reply:
We've always had it extracted. It is just now it has changed to being extracted with ZAPD rather than using a GLOBAL_ASM and including an asm file that contained the rodata.